autogpts/forge/forge/components/web/selenium.py

```diff
@@ -49,6 +49,10 @@ class BrowsingError(CommandExecutionError):
     """An error occurred while trying to browse the page"""
 
 
+class TooMuchOutputError(CommandExecutionError):
+    """The operation generated more output than what the Agent can process"""
+
+
```

Comment:
IMO this is a general error and shouldn't be tied to this specific component.

Reply:
Moved back to `exceptions.py`